storehaus-mysql/src/main/scala/com/twitter/storehaus/mysql/ValueMapper.scala

```diff
@@ -78,7 +78,8 @@ object ValueMapper {
       case EmptyValue => Some("")
       case NullValue => None
       // all other types are currently unsupported
-      case _ => throw new UnsupportedOperationException(v.getClass.getName + " is currently not supported.")
+      case _ => throw new UnsupportedOperationException(value.getClass.getName +
```

Comment:
can we do `case other => ` and include `$other` in the error message? Helps debugging usually.